eng/config/app.manifest

```diff
@@ -14,6 +14,7 @@
   <application xmlns="urn:schemas-microsoft-com:asm.v3">
     <windowsSettings>
       <longPathAware xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">true</longPathAware>
+      <supportedArchitectures xmlns="http://schemas.microsoft.com/SMI/2024/WindowsSettings">amd64 arm64</supportedArchitectures>
```

Comment:
I assume x86 works by default?

Reply:
No idea, the manifest supports only these two values.

Reply:
Can't merge with the answer being "no idea". Need to know answers to the x86 question.

Reply:
Got confirmation that this is a no-op on platforms that don't support this new part of the manifest.

Reply:
@jaredpar Is it not a problem because any Windows new enough to understand this doesn't have x86 builds anymore?

Reply:
This section is just stating that this binary _can_ run on amd64 and arm64. It's not saying that it can only run on those.